src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -175,7 +175,9 @@ private bool IsBaseTypeWithKnownTypesDefined(DataContract dataContract, out IEnu
 
             if (dataContract.DataType != DataType.Object) return false;
 
-            var subTypes = _generatorOptions.SubTypesSelector(dataContract.UnderlyingType);
+            var subTypes = _generatorOptions
+                .SubTypesSelector(dataContract.UnderlyingType)
+                .Where(type => !type.IsGenericTypeDefinition); ;
```

Comment:
Could you add a comment explaining why generic types are skipped?

I'm new to the internals of the codebase and it isn't immediately clear to me.